torch/csrc/jit/codegen/cuda/arith.cpp

```diff
@@ -602,7 +602,8 @@ TensorView* reductionOp(
     const std::vector<int>& axes,
     Val* init,
     TensorView* tv,
-    bool keep_dim /*=false*/) {
+    bool keep_dim /*=false*/,
+    DataType accumulator_type /*=Null*/) {
```

Comment:
a minor reduction op change to support `fp16` reduction input and `fp32` accumulator output.